CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+#### Changes to the Stake Snapshot
+
+Since the stake snapshot changes at each epoch boundary, a new current voting tally must be calculated for each governance action based on the votes that have been cast before any new votes are counted.  This avoids the same stake being used two or more times.  _This new tally may result in immediate ratification of a governance action._
+
+### Delegated Representatives (DReps)
+
+> **Warning**
+> The design of the DReps is still subject to change and should not be conflated with Project Catalyst DReps.
+
+Existing stake keys will be given new responsibilities. They will be able to delegate their stake to DReps. DRep Registration will mimic the existing stake delegation mechanisms (via certificates).
+
+The following new types of voting certificates will be added:
+
+* DRep registration certificate. Including:
+  * DRep ID: hash of verification key
+  * a URL for any metadata that is relevant to the DRep
+  * a hash of the contents of this URL
+
+* DRep retirement certificate. Including:
+  * DRep ID
+  * epoch number after which the DRep will retire
+
+* Vote delegation certificate. Including:
+  * stake credential
+  * DRep ID
+
+// TODO: Provide CBOR specification in the annexe for those new certificates.
+
+The authorization scheme (i.e. which signatures are required) mimics the existing stake delegation certificate authorization scheme.
+
+#### New Stake Distribution for DReps
+
+In addition to the existing per-stake-credential distribution and the per-stake-pool distribution, we will now have a new per-DRep stake distribution.
+
+This distribution will determine how much stake is backed by each `yes` (or `no`) vote from a DRep. The exact stake snapshot used for block production by the SPOs will also be used for the DReps for voting.
+
+#### Definitions surrounding Voting Stake
+
+We define some notions around voting stake:
+
+* Lovelace contained in a transaction output is considered **active for voting** if:
+  * It contains a registered stake credential.
+  * The stake credential has delegated its voting rights to a registered DRep.
+* Relative to some given percentage `P`, we say that there is **sufficient active voting stake** if
+  the ratio of the *total active voting stake* to the *total stake in circulation* is at least `P`.
+* Relative to a percentage `P`, a DRep (SPO) **vote threshold has been met** if the sum of the relative stake controlled by the DReps (SPOs) voting 'yes' to a governance action minus the sum of the relative stake controlled by the DReps (SPOs) voting 'no' is at least `P`.
+
+> **Note**
+> There are several alternative definitions for "the total stake in circulation":
+>  1. The sum of the UTxO and the rewards accounts.
+>  2. The sum of the UTxO, the rewards accounts, the fee pot, and the deposit pot.
+>  3. The total ADA supply (i.e. 45 billion ADA) minus the reserves[^3].
+>  We leave the choice open for discussion.
+
+## Rationale
```

Comment:
I would love to see Rationale sections for:
1. The choices of which conditions for which governance actions are required, e.g. why does initiating a HF not require an AVST threshold? All of these seem very significant.
2. The intended effect of the AVST thresholds in general
3. The use of a voting threshold based on the difference between yes and no votes versus a simple yes threshold
4. The choices around metadata
5. The decisions to have separate governance actions for treasury withdrawals of various

Reply:
fantastic suggestions, thank you! I will add these shortly.

Reply:
Hi. I would love to contribute to the developement of the Governace discussion. I am a new GitHubber and not a coder. How can I submit my ideas?

Reply:
welcome @WhiteHorseAvatar ! posting comments like you've done here is great.

Reply:
Thanks Jared. This is a very important conversation and I am keen to read everyones ideas and exercise some of my own. As a passionate and hopefull Cardano believer and thinker I see a possibilty of all humanity thriving on the blockchain .